### ISSUE TITLE
Qwen2.5-VL based generate() returns NoneType logits

### ISSUE THREAD

**Issue by jeongdaeyun** [YXB-NKU/SE-GUI#8]
Thank you for your amazing research and for sharing the model!

I loaded the SE-GUI 7B model from Hugging Face and followed the README on this page, as well as the microsoft/GUI-Actor repository. However, when I try to use `model.generate()`, I encounter the following issue:

I’m using Qwen2.5-VL as the backbone, and logits is returned as None. When I tried to retrieve the logits, the model output was <image end> instead of coordinates.

Is there something I might be doing wrong? I would really appreciate it if you could let me know what I may have missed.

I look forward to your response. Thank you very much!

**Comment by YXB-NKU** [comment]
Hello!Thanks for your interest in our work!
i have a question , why you follow the readme as well as microsoft/GUI-Actor? Our evaluation code is under src/eval.
